testTransactionTaskSweeping fails in master

**Problem description**
Here is the stack trace:

```
com.emc.pravega.controller.task.TaskTest > testTransactionTaskSweeping FAILED
    java.lang.AssertionError: expected:<COMMITTING> but was:<ABORTING>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:118)
        at org.junit.Assert.assertEquals(Assert.java:144)
        at com.emc.pravega.controller.task.TaskTest.testTransactionTaskSweeping(TaskTest.java:350)
```

**Problem location**
Controller tests.

--- a/controller/server/src/test/java/com/emc/pravega/controller/task/TaskTest.java
+++ b/controller/server/src/test/java/com/emc/pravega/controller/task/TaskTest.java
@@ -341,9 +341,14 @@ public class TaskTest {
         // Ensure that a transaction is created.
         map = streamStore.getActiveTxns(SCOPE, stream1, null, executor).join();
         assertEquals(2, map.size());
-        txId = map.keySet().iterator().next();
+        Optional<UUID> txIdOpt = map.entrySet().stream()
+                .filter(e -> e.getValue().getTxnStatus() == TxnStatus.OPEN)
+                .map(e -> e.getKey())
+                .findAny();
+        Assert.assertTrue(txIdOpt.isPresent());
 
         // Commit the transaction.
+        txId = txIdOpt.get();
         completePartialTask(mockTxnTasks.commitTxn(SCOPE, stream1, txId, null), deadHost, sweeper);
         // Ensure that transaction state is COMMITTING.
         status = streamStore.getTransactionData(SCOPE, stream1, txId, null, executor).join().getStatus();

